Projects/GLU/GLU07/GLU07-AWS100.md

```diff
@@ -0,0 +1,54 @@
+
+# GLU07-AWS100 - Setup a simple state machine with at least 2 steps
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+- Level 100 (Introductory)
+
+## Estimated Time
+- 60 minutes 
+
+## Estimated Cost
+- The Step Functions free tier includes 4,000 free state transitions per month. After the free tier exhausts you pay $0.025 per 1,000 state transitions. See Details [here](https://aws.amazon.com/step-functions/pricing/)
+- The AWS Lambda free usage tier includes **1M free requests per month and 400,000 GB-seconds of compute time per month**
+
```

Comment:
Is this project included in the free tier overall? Lot of components, can you just mention `This project is included in the free tier` please if it is?